Apps/phgsgmail/gsgmail_connector.py

```diff
@@ -243,6 +248,99 @@ def _map_email_details(self, input_email):
 
         return (phantom.APP_SUCCESS, input_email)
 
+    def _get_email_headers_from_part(self, part, charset=None):
+
+        email_headers = list(part.items())
+
+        # TODO: the next 2 ifs can be condensed to use 'or'
```

Comment:
Should this TODO be completed?

Reply:
think this code was copied from the EWS app? See above, but this was a PR from the PS team